lib/spree/adyen/api_response.rb

```diff
@@ -38,6 +42,11 @@ def authorisation_response?
         @gateway_response.is_a?(::Adyen::REST::AuthorisePayment::Response)
       end
 
+      def authorisation_redirect_response?
+        @gateway_response.is_a?(::Adyen::REST::AuthorisePayment::Response) &&
+          @gateway_response.attributes["paymentResult.resultCode"] == "RedirectShopper"
+      end
```

Comment:
I'm not sure this extra method is required. This logic could just live in `redirect?`. (Or we could just add an alias.)